src/ralph/back_office/admin.py

```diff
@@ -79,6 +81,9 @@ class BackOfficeAssetAdmin(
         'status', 'barcode', 'purchase_order', 'model', 'user', 'warehouse',
         'sn', 'hostname', 'invoice_date', 'invoice_no', 'region',
     ]
+    multiadd_fields = ['sn', 'barcode']
```

Comment:
what about imei? as far as i remember there were imei field too when specific category was chosen for bo asset. Maybe you could call `get_multiadd_fields` method, to which new object will be passed and by default it'll return value of `multiadd_fields` attribute, but you could customize it for BO Asset and depending on asset category add (or not) imei to multiadd fields. @ar4s @xliiv what do you think?